index.js

```diff
@@ -292,16 +293,28 @@ const app = express()
 app.use(express.json())
 app.use(cors({ origin: '*' }))
 
+// used for very simple caching of the feed
+const feedCacheTimeLimit = ONE_MINUTE_MILLIS / 6 // 10 seconds
```

Comment:
Cache can be 1 minute since that's roughly the Tezos block-time so nothing can happen every 10 secs

Reply:
Oh that's even better then.

There is better ways to do caching for sure, but wanted to make the least amount of code changes for the biggest win at the moment.

This could easily be removed once a better cache mechanism has been implemented, but as I said, will be big win.

I'm still trying to understand the API code itself.